packages/tools/client-debugger/client-debugger-view/src/DevtoolsView.tsx

```diff
@@ -438,10 +440,37 @@ function ContainersMenuSection(props: ContainersMenuSectionProps): React.ReactEl
 		);
 	}
 
-	// TODO: add button to refresh list of containers
 	return (
-		<MenuSection header="Containers" key="container-selection-menu-section">
-			{containerSectionInnerView}
-		</MenuSection>
+		<div>
```

Comment:
I would probably use a <Stack horizontal> here, rather than a div. And wrap each of the children in a <Stack.Item>. That will give use consistent styling with other places we use that sort of layout.